iOS/CodePush.h

```diff
@@ -104,6 +104,24 @@ failCallback:(void (^)(NSError *err))failCallback;
 
 @end
 
+@interface CodePushUtils : NSObject
```

Comment:
Should we call this `CodePushUpdateUtils`? The current name implies it is more generic than it actually is.